pdns/pdns_recursor.cc

```diff
@@ -5057,15 +5057,15 @@ static int serviceMain(int argc, char*argv[])
 
   g_useKernelTimestamp = ::arg().mustDo("protobuf-use-kernel-timestamp");
 
-  blacklistStats(StatComponent::API, ::arg()["stats-api-blacklist"]);
-  blacklistStats(StatComponent::Carbon, ::arg()["stats-carbon-blacklist"]);
-  blacklistStats(StatComponent::RecControl, ::arg()["stats-rec-control-blacklist"]);
-  blacklistStats(StatComponent::SNMP, ::arg()["stats-snmp-blacklist"]);
+  disableStats(StatComponent::API, ::arg()["stats-api-blacklist"]);
```

Comment:
this still uses the old names of the configs. Perhaps we should merge their values into the new one?

Reply:
`disableStats` is both called for the old and new names, see below.

4.6.0 wil warn about them and 4.7 will remove the odl names.

Reply:
excellent